esd_services_api_client/beast/v3/_auth.py

```diff
@@ -0,0 +1,87 @@
+"""
+ Beast Auth class
+ Based on https://docs.python-requests.org/en/master/user/advanced/#custom-authentication
+"""
+#  Copyright (c) 2023. ECCO Sneaks & Data
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from datetime import datetime
+from typing import Optional
+
+from requests.auth import AuthBase
+
+from azure.identity import DefaultAzureCredential
+
+
+class BeastAuth(AuthBase):
```

Comment:
I don't think we still need it, It should be removed in beast v2, isn't it?

Reply:
yeah, removing